Assets/Scripts/Utilities/RenderSettingsManager.cs

```diff
@@ -46,6 +44,15 @@ void Awake()
 
 	void Update()
 	{
+		dayCycleTimer += Time.deltaTime;
+
+		timeOfDay = (-Mathf.Cos(dayCycleTimer/dayCycleTime * Mathf.Rad2Deg) * 0.5f) + 0.5f;
```

Comment:
Also leave a comment specifying the range of possible values for timeOfDay. It takes a bit to figure out if you're looking at it for the first time.